stdlib/2and3/mmap.pyi

```diff
@@ -56,7 +63,15 @@ if sys.version_info >= (3,):
         closed: bool
         if sys.version_info >= (3, 8) and sys.platform != "win32":
             def madvise(self, option: int, start: int = ..., length: int = ...) -> None: ...
-        def rfind(self, sub: bytes, start: int = ..., stop: int = ...) -> int: ...
+        if sys.version_info >= (3, 5):
+            def find(self, sub: ReadableBuffer, start: int = ..., stop: int = ...) -> int: ...
+            def rfind(self, sub: ReadableBuffer, start: int = ..., stop: int = ...) -> int: ...
+        else:
+            def rfind(self, sub: bytes, start: int = ..., stop: int = ...) -> int: ...
+        if sys.version_info >= (3, 6):
```

Comment:
Since typeshed only supports Python 2.7 and 3.5+, you can leave out the version check and the else branch.

Reply:
And the mark is wrong, this is for lines 66 to 70.